doc/man3/OSSL_DECODER_CTX_new_for_pkey.pod

```diff
@@ -14,11 +14,11 @@ OSSL_DECODER_CTX_set_passphrase_cb
  #include <openssl/decoder.h>
 
  OSSL_DECODER_CTX *
- OSSL_DECODER_CTX_new_by_EVP_PKEY(EVP_PKEY **pkey,
-                                  const char *input_type,
-                                  const char *input_struct,
-                                  const char *keytype, int selection,
-                                  OSSL_LIB_CTX *libctx, const char *propquery);
+ OSSL_DECODER_CTX_new_for_pkey(EVP_PKEY **pkey,
```

Comment:
I think this doc needs to explain what I<pkey> is.

Reply:
I can add that although strictly speaking it is off topic for the PR.

Reply:
PTAL

Reply:
> PTAL

I had to [look that up!](https://en.wiktionary.org/wiki/PTAL)